hp/src/CodeGenerator.hs

```diff
@@ -232,7 +232,14 @@ generateBoolExpression (BoolValue b) = generateBoolValue b
 generateBoolExpression (BoolComparison c) = generateComparison c
 
 generateBoolComparator :: BoolComparator -> String
-generateBoolComparator bc = ""
+generateBoolComparator Eq  = "=="
```

Comment:
Please use strict equality, I know our grammar looks like this "==", but we are transpiling it to js and a strict comparison needs to be done.

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness)